spec/features/collection_templates/template_edit_spec.rb

```diff
@@ -0,0 +1,38 @@
+# Tests for verifying the code changes involved in editing a template.
+# The sparsity of these tests is because the underlying functionality
+# of editing a template is assumed to be valid if the tests for collection draft
+# editing are valid.  As such, only the template name changing and validation
+# are currently tested.
+
+describe 'While editing a collection template', js:true do
+  before do
+    login
+    create(:full_collection_template, template_name: 'An Example Template')
+  end
+
+  context 'when editing a draft' do
+    before do
+      draft = create(:full_collection_template, template_name: 'An Example Template2')
+      visit edit_collection_template_path(draft)
+    end
+
+    it 'provides error validation for template names' do
+      expect(page).to have_content('An Example Template2')
+      fill_in 'Template Name', with: 'An Example Template'
+      # Need to do something else to trigger the validation because the fill_in
```

Comment:
You can always just find another element and click on it, I have had to do that for other tests